### DISCUSSION TITLE
不知道为啥 最近我的世界玩着玩着老是要崩 求大佬帮忙看看

### DISCUSSION THREAD

**Discussion by lmxly** [Meloong-Git/PCL#1673] [提问] [answered]
[错误报告-2023-4-17_17.56.44.zip](https://github.com/Hex-Dragon/PCL2/files/11252481/-2023-4-17_17.56.44.zip)

**Comment by WorldHim** [comment, marked as answer]
`[Render thread/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='Error has been generated. GL error GL_INVALID_OPERATION in (null): (ID: 173538523) Generic error'`
渲染问题，建议打开OpenGL错误显示，然后尝试调低画质，关闭光影与材质或其他选项等，确认问题是否继续发生。

---
Upd: 也有可能是显卡驱动或硬件问题，若仍无法解决可以尝试更换软硬件。

**Reply by lmxly** [reply]
我去试试 谢谢大佬

**Reply by iPodtouch6557** [reply]
之前有一个OpenGL错误，错误代码忘了，像是4xxx，反正结果是铁傀儡的渲染皮肤糊得像 #1656 ，但是结构是完整的，攻击也正常，~我甚至还从铁傀儡那边白嫖了五个铁~

**Comment by Pigeon0v0** [comment]
也许可以尝试重装显卡驱动？
